### ISSUE TITLE
Running `luarocks` without arguments on windows right after installation causes the program to crash

### ISSUE THREAD

**Issue by echiesse** [luarocks/luarocks#1418]
Platform: Windows.
LuaRocks version: 3.9.0 (64bit)
Verbose Output:
```
 λ luarocks --verbose
fs.current_dir()
fs.current_dir()
fs.is_superuser()
fs.is_file("c:/external/cyglua5.4.dll")
fs.absolute_name("c:/external/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lib/cyglua5.4.dll")
fs.absolute_name("c:/external/lib/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/bin/cyglua5.4.dll")
fs.absolute_name("c:/external/bin/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/liblua5.4.dll")
fs.absolute_name("c:/external/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lib/liblua5.4.dll")
fs.absolute_name("c:/external/lib/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/bin/liblua5.4.dll")
fs.absolute_name("c:/external/bin/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lua5.4.dll")
fs.absolute_name("c:/external/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lib/lua5.4.dll")
fs.absolute_name("c:/external/lib/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/bin/lua5.4.dll")
fs.absolute_name("c:/external/bin/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lua5.4.lib")
fs.absolute_name("c:/external/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/external/lib/lua5.4.lib")
fs.absolute_name("c:/external/lib/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/external/bin/lua5.4.lib")
fs.absolute_name("c:/external/bin/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/external/cyglua54.dll")
fs.absolute_name("c:/external/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/external/lib/cyglua54.dll")
fs.absolute_name("c:/external/lib/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/external/bin/cyglua54.dll")
fs.absolute_name("c:/external/bin/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/external/liblua54.dll")
fs.absolute_name("c:/external/liblua54.dll")
fs.current_dir()
fs.is_file("c:/external/lib/liblua54.dll")
fs.absolute_name("c:/external/lib/liblua54.dll")
fs.current_dir()
fs.is_file("c:/external/bin/liblua54.dll")
fs.absolute_name("c:/external/bin/liblua54.dll")
fs.current_dir()
fs.is_file("c:/external/lua54.dll")
fs.absolute_name("c:/external/lua54.dll")
fs.current_dir()
fs.is_file("c:/external/lib/lua54.dll")
fs.absolute_name("c:/external/lib/lua54.dll")
fs.current_dir()
fs.is_file("c:/external/bin/lua54.dll")
fs.absolute_name("c:/external/bin/lua54.dll")
fs.current_dir()
fs.is_file("c:/external/lua54.lib")
fs.absolute_name("c:/external/lua54.lib")
fs.current_dir()
fs.is_file("c:/external/lib/lua54.lib")
fs.absolute_name("c:/external/lib/lua54.lib")
fs.current_dir()
fs.is_file("c:/external/bin/lua54.lib")
fs.absolute_name("c:/external/bin/lua54.lib")
fs.current_dir()
fs.is_file("c:/external/cyglua-5.4.dll")
fs.absolute_name("c:/external/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lib/cyglua-5.4.dll")
fs.absolute_name("c:/external/lib/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/bin/cyglua-5.4.dll")
fs.absolute_name("c:/external/bin/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/liblua-5.4.dll")
fs.absolute_name("c:/external/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lib/liblua-5.4.dll")
fs.absolute_name("c:/external/lib/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/bin/liblua-5.4.dll")
fs.absolute_name("c:/external/bin/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lua-5.4.dll")
fs.absolute_name("c:/external/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lib/lua-5.4.dll")
fs.absolute_name("c:/external/lib/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/bin/lua-5.4.dll")
fs.absolute_name("c:/external/bin/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/external/lua-5.4.lib")
fs.absolute_name("c:/external/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/external/lib/lua-5.4.lib")
fs.absolute_name("c:/external/lib/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/external/bin/lua-5.4.lib")
fs.absolute_name("c:/external/bin/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/external/cyglua-54.dll")
fs.absolute_name("c:/external/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/external/lib/cyglua-54.dll")
fs.absolute_name("c:/external/lib/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/external/bin/cyglua-54.dll")
fs.absolute_name("c:/external/bin/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/external/liblua-54.dll")
fs.absolute_name("c:/external/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/external/lib/liblua-54.dll")
fs.absolute_name("c:/external/lib/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/external/bin/liblua-54.dll")
fs.absolute_name("c:/external/bin/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/external/lua-54.dll")
fs.absolute_name("c:/external/lua-54.dll")
fs.current_dir()
fs.is_file("c:/external/lib/lua-54.dll")
fs.absolute_name("c:/external/lib/lua-54.dll")
fs.current_dir()
fs.is_file("c:/external/bin/lua-54.dll")
fs.absolute_name("c:/external/bin/lua-54.dll")
fs.current_dir()
fs.is_file("c:/external/lua-54.lib")
fs.absolute_name("c:/external/lua-54.lib")
fs.current_dir()
fs.is_file("c:/external/lib/lua-54.lib")
fs.absolute_name("c:/external/lib/lua-54.lib")
fs.current_dir()
fs.is_file("c:/external/bin/lua-54.lib")
fs.absolute_name("c:/external/bin/lua-54.lib")
fs.current_dir()
fs.is_file("c:/external/cyglua.dll")
fs.absolute_name("c:/external/cyglua.dll")
fs.current_dir()
fs.is_file("c:/external/lib/cyglua.dll")
fs.absolute_name("c:/external/lib/cyglua.dll")
fs.current_dir()
fs.is_file("c:/external/bin/cyglua.dll")
fs.absolute_name("c:/external/bin/cyglua.dll")
fs.current_dir()
fs.is_file("c:/external/liblua.dll")
fs.absolute_name("c:/external/liblua.dll")
fs.current_dir()
fs.is_file("c:/external/lib/liblua.dll")
fs.absolute_name("c:/external/lib/liblua.dll")
fs.current_dir()
fs.is_file("c:/external/bin/liblua.dll")
fs.absolute_name("c:/external/bin/liblua.dll")
fs.current_dir()
fs.is_file("c:/external/lua.dll")
fs.absolute_name("c:/external/lua.dll")
fs.current_dir()
fs.is_file("c:/external/lib/lua.dll")
fs.absolute_name("c:/external/lib/lua.dll")
fs.current_dir()
fs.is_file("c:/external/bin/lua.dll")
fs.absolute_name("c:/external/bin/lua.dll")
fs.current_dir()
fs.is_file("c:/external/lua.lib")
fs.absolute_name("c:/external/lua.lib")
fs.current_dir()
fs.is_file("c:/external/lib/lua.lib")
fs.absolute_name("c:/external/lib/lua.lib")
fs.current_dir()
fs.is_file("c:/external/bin/lua.lib")
fs.absolute_name("c:/external/bin/lua.lib")
fs.current_dir()
fs.is_file("c:/mingw/cyglua5.4.dll")
fs.absolute_name("c:/mingw/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/cyglua5.4.dll")
fs.absolute_name("c:/mingw/lib/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/cyglua5.4.dll")
fs.absolute_name("c:/mingw/bin/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/liblua5.4.dll")
fs.absolute_name("c:/mingw/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/liblua5.4.dll")
fs.absolute_name("c:/mingw/lib/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/liblua5.4.dll")
fs.absolute_name("c:/mingw/bin/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua5.4.dll")
fs.absolute_name("c:/mingw/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua5.4.dll")
fs.absolute_name("c:/mingw/lib/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua5.4.dll")
fs.absolute_name("c:/mingw/bin/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua5.4.lib")
fs.absolute_name("c:/mingw/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua5.4.lib")
fs.absolute_name("c:/mingw/lib/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua5.4.lib")
fs.absolute_name("c:/mingw/bin/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/mingw/cyglua54.dll")
fs.absolute_name("c:/mingw/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/cyglua54.dll")
fs.absolute_name("c:/mingw/lib/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/cyglua54.dll")
fs.absolute_name("c:/mingw/bin/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/liblua54.dll")
fs.absolute_name("c:/mingw/liblua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/liblua54.dll")
fs.absolute_name("c:/mingw/lib/liblua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/liblua54.dll")
fs.absolute_name("c:/mingw/bin/liblua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua54.dll")
fs.absolute_name("c:/mingw/lua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua54.dll")
fs.absolute_name("c:/mingw/lib/lua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua54.dll")
fs.absolute_name("c:/mingw/bin/lua54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua54.lib")
fs.absolute_name("c:/mingw/lua54.lib")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua54.lib")
fs.absolute_name("c:/mingw/lib/lua54.lib")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua54.lib")
fs.absolute_name("c:/mingw/bin/lua54.lib")
fs.current_dir()
fs.is_file("c:/mingw/cyglua-5.4.dll")
fs.absolute_name("c:/mingw/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/cyglua-5.4.dll")
fs.absolute_name("c:/mingw/lib/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/cyglua-5.4.dll")
fs.absolute_name("c:/mingw/bin/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/liblua-5.4.dll")
fs.absolute_name("c:/mingw/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/liblua-5.4.dll")
fs.absolute_name("c:/mingw/lib/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/liblua-5.4.dll")
fs.absolute_name("c:/mingw/bin/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua-5.4.dll")
fs.absolute_name("c:/mingw/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua-5.4.dll")
fs.absolute_name("c:/mingw/lib/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua-5.4.dll")
fs.absolute_name("c:/mingw/bin/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua-5.4.lib")
fs.absolute_name("c:/mingw/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua-5.4.lib")
fs.absolute_name("c:/mingw/lib/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua-5.4.lib")
fs.absolute_name("c:/mingw/bin/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/mingw/cyglua-54.dll")
fs.absolute_name("c:/mingw/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/cyglua-54.dll")
fs.absolute_name("c:/mingw/lib/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/cyglua-54.dll")
fs.absolute_name("c:/mingw/bin/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/liblua-54.dll")
fs.absolute_name("c:/mingw/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/liblua-54.dll")
fs.absolute_name("c:/mingw/lib/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/liblua-54.dll")
fs.absolute_name("c:/mingw/bin/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua-54.dll")
fs.absolute_name("c:/mingw/lua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua-54.dll")
fs.absolute_name("c:/mingw/lib/lua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua-54.dll")
fs.absolute_name("c:/mingw/bin/lua-54.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua-54.lib")
fs.absolute_name("c:/mingw/lua-54.lib")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua-54.lib")
fs.absolute_name("c:/mingw/lib/lua-54.lib")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua-54.lib")
fs.absolute_name("c:/mingw/bin/lua-54.lib")
fs.current_dir()
fs.is_file("c:/mingw/cyglua.dll")
fs.absolute_name("c:/mingw/cyglua.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/cyglua.dll")
fs.absolute_name("c:/mingw/lib/cyglua.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/cyglua.dll")
fs.absolute_name("c:/mingw/bin/cyglua.dll")
fs.current_dir()
fs.is_file("c:/mingw/liblua.dll")
fs.absolute_name("c:/mingw/liblua.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/liblua.dll")
fs.absolute_name("c:/mingw/lib/liblua.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/liblua.dll")
fs.absolute_name("c:/mingw/bin/liblua.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua.dll")
fs.absolute_name("c:/mingw/lua.dll")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua.dll")
fs.absolute_name("c:/mingw/lib/lua.dll")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua.dll")
fs.absolute_name("c:/mingw/bin/lua.dll")
fs.current_dir()
fs.is_file("c:/mingw/lua.lib")
fs.absolute_name("c:/mingw/lua.lib")
fs.current_dir()
fs.is_file("c:/mingw/lib/lua.lib")
fs.absolute_name("c:/mingw/lib/lua.lib")
fs.current_dir()
fs.is_file("c:/mingw/bin/lua.lib")
fs.absolute_name("c:/mingw/bin/lua.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/cyglua5.4.dll")
fs.absolute_name("c:/windows/system32/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/cyglua5.4.dll")
fs.absolute_name("c:/windows/system32/lib/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/cyglua5.4.dll")
fs.absolute_name("c:/windows/system32/bin/cyglua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/liblua5.4.dll")
fs.absolute_name("c:/windows/system32/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/liblua5.4.dll")
fs.absolute_name("c:/windows/system32/lib/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/liblua5.4.dll")
fs.absolute_name("c:/windows/system32/bin/liblua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua5.4.dll")
fs.absolute_name("c:/windows/system32/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua5.4.dll")
fs.absolute_name("c:/windows/system32/lib/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua5.4.dll")
fs.absolute_name("c:/windows/system32/bin/lua5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua5.4.lib")
fs.absolute_name("c:/windows/system32/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua5.4.lib")
fs.absolute_name("c:/windows/system32/lib/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua5.4.lib")
fs.absolute_name("c:/windows/system32/bin/lua5.4.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/cyglua54.dll")
fs.absolute_name("c:/windows/system32/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/cyglua54.dll")
fs.absolute_name("c:/windows/system32/lib/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/cyglua54.dll")
fs.absolute_name("c:/windows/system32/bin/cyglua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/liblua54.dll")
fs.absolute_name("c:/windows/system32/liblua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/liblua54.dll")
fs.absolute_name("c:/windows/system32/lib/liblua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/liblua54.dll")
fs.absolute_name("c:/windows/system32/bin/liblua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua54.dll")
fs.absolute_name("c:/windows/system32/lua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua54.dll")
fs.absolute_name("c:/windows/system32/lib/lua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua54.dll")
fs.absolute_name("c:/windows/system32/bin/lua54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua54.lib")
fs.absolute_name("c:/windows/system32/lua54.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua54.lib")
fs.absolute_name("c:/windows/system32/lib/lua54.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua54.lib")
fs.absolute_name("c:/windows/system32/bin/lua54.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/cyglua-5.4.dll")
fs.absolute_name("c:/windows/system32/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/cyglua-5.4.dll")
fs.absolute_name("c:/windows/system32/lib/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/cyglua-5.4.dll")
fs.absolute_name("c:/windows/system32/bin/cyglua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/liblua-5.4.dll")
fs.absolute_name("c:/windows/system32/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/liblua-5.4.dll")
fs.absolute_name("c:/windows/system32/lib/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/liblua-5.4.dll")
fs.absolute_name("c:/windows/system32/bin/liblua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua-5.4.dll")
fs.absolute_name("c:/windows/system32/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua-5.4.dll")
fs.absolute_name("c:/windows/system32/lib/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua-5.4.dll")
fs.absolute_name("c:/windows/system32/bin/lua-5.4.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua-5.4.lib")
fs.absolute_name("c:/windows/system32/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua-5.4.lib")
fs.absolute_name("c:/windows/system32/lib/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua-5.4.lib")
fs.absolute_name("c:/windows/system32/bin/lua-5.4.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/cyglua-54.dll")
fs.absolute_name("c:/windows/system32/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/cyglua-54.dll")
fs.absolute_name("c:/windows/system32/lib/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/cyglua-54.dll")
fs.absolute_name("c:/windows/system32/bin/cyglua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/liblua-54.dll")
fs.absolute_name("c:/windows/system32/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/liblua-54.dll")
fs.absolute_name("c:/windows/system32/lib/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/liblua-54.dll")
fs.absolute_name("c:/windows/system32/bin/liblua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua-54.dll")
fs.absolute_name("c:/windows/system32/lua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua-54.dll")
fs.absolute_name("c:/windows/system32/lib/lua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua-54.dll")
fs.absolute_name("c:/windows/system32/bin/lua-54.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua-54.lib")
fs.absolute_name("c:/windows/system32/lua-54.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua-54.lib")
fs.absolute_name("c:/windows/system32/lib/lua-54.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua-54.lib")
fs.absolute_name("c:/windows/system32/bin/lua-54.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/cyglua.dll")
fs.absolute_name("c:/windows/system32/cyglua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/cyglua.dll")
fs.absolute_name("c:/windows/system32/lib/cyglua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/cyglua.dll")
fs.absolute_name("c:/windows/system32/bin/cyglua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/liblua.dll")
fs.absolute_name("c:/windows/system32/liblua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/liblua.dll")
fs.absolute_name("c:/windows/system32/lib/liblua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/liblua.dll")
fs.absolute_name("c:/windows/system32/bin/liblua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua.dll")
fs.absolute_name("c:/windows/system32/lua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua.dll")
fs.absolute_name("c:/windows/system32/lib/lua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua.dll")
fs.absolute_name("c:/windows/system32/bin/lua.dll")
fs.current_dir()
fs.is_file("c:/windows/system32/lua.lib")
fs.absolute_name("c:/windows/system32/lua.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/lib/lua.lib")
fs.absolute_name("c:/windows/system32/lib/lua.lib")
fs.current_dir()
fs.is_file("c:/windows/system32/bin/lua.lib")
fs.absolute_name("c:/windows/system32/bin/lua.lib")
fs.current_dir()
fs.exists("C:\\Users\\xxxxx\\bin")
fs.exists(nil)
```
## Description:
I just ran the command
```
> luarocks
```
right after putting the executable in the path as instructed in the documentation. 
Then the message below with an exception pops up:
![image](https://user-images.githubusercontent.com/5909008/168931754-5f0c3f8d-1c31-465e-8705-02a8104e82a9.png)

**Comment by hishamhm** [comment]
@echiesse Thanks for the report! Just to confirm, this is using the provided binary, right? (`luarocks.exe`)

**Comment by hishamhm** [comment]
Please let me know if these binaries fix the issue! Thanks!
https://hisham.hm/tmp/luarocks-3.9.0-fix-windows-64.zip

**Comment by MathiasSven** [comment]
I had just installed the latest windows binaries from luarocks/releases and was having a similar issue in that running `luarocks.exe` would give me the same popup as OP, this new `fix` binary solved my issue.

**Comment by echiesse** [comment]
Tested here and it works for what I reported. Thank you.
---
Eric


Em qua., 18 de mai. de 2022 às 00:24, Hisham Muhammad <
***@***.***> escreveu:

**Comment by hishamhm** [comment]
Thank you all for the feedback. I published LuaRocks 3.9.1, the new binaries shouldn't have the problem. But let me know if anything's wrong!
